.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,41 @@
+name: Coding Standards
+
+on:
+  push:
+    branches: [ $default-branch ]
```

Comment:
- Please change into `branches: master`. It seems that the `$default_branch` is used got the template. Not GitHub action. Please see [this](https://stackoverflow.com/questions/64781462/github-actions-default-branch-variable) to know details.